docs/database/migrations.mdx

```diff
@@ -57,6 +57,36 @@ you need to do is pass the `req` object to any [local API](/docs/local-api/overv
 after your `up` or `down` function runs. If the migration errors at any point or fails to commit, it is caught and the
 transaction gets aborted. This way no change is made to the database if the migration fails.
 
+### Using database directly with the transaction
+
+Additionally, you can bypass Payload's layer entirely and perform operations directly on your underlying database within the active transaction
+
+### MongoDB:
+```ts
+import { type MigrateUpArgs } from '@payloadcms/db-mongodb'
+
+export async function up({ session, payload, req }: MigrateUpArgs): Promise<void> {
+  const posts = await payload.db.collections.posts.collection.find({ session }).toArray()
+}
+```
+### Postgres:
+```ts
+import { type MigrateUpArgs, sql } from '@payloadcms/db-postgres'
+
+export async function up({ db, payload, req }: MigrateUpArgs): Promise<void> {
+  const { rows: posts } = await db.execute(sql`SELECT * from posts`)
+}
+```
+
+### SQLite:
```

Comment:
SQLite transactions are opt-in only because they weren't working in my experience like Postgres and MongoDB. Is this misleading since they need to do more to configure SQLite?

Reply:
> SQLite transactions are opt-in

I didn't find any mentioning of this in docs. Added as well to `overview/transactions` like with mongodb.